download_script.py

```diff
@@ -0,0 +1,50 @@
+import requests
+import argparse
+
+# help function for better understanding of the scripts functionality
+
+description = """  This script is used to download all the pickle/dataset files needed to train, evaluate and predict using the model.
+                    Also downloads model tokenizers used for preprocessing the datasets before feeding them to the model \n
+
+                   The script assumes to be run at spambrainz_ml top directory \n
+
+                   The files if present are overwritten by the script, kindly be careful"""
+
+parser = argparse.ArgumentParser(description=description)
+
+args = parser.parse_args()
+
+# Tokenizer files reqeuried for preprocessing.py file
+bio_tokenizer_url = 'https://github.com/metabrainz/spambrainz_ml/releases/download/v-0.1/bio_tokenizer.pickle'
+web_tokenizer_url = 'https://github.com/metabrainz/spambrainz_ml/releases/download/v-0.1/web_tokenizer.pickle'
+email_tokenizer_url = 'https://github.com/metabrainz/spambrainz_ml/releases/download/v-0.1/email_tokenizer.pickle'
+
+
```

Comment:
Please drop this extraneous blank line and trailing blank lines at the end of the script.

```suggestion

```